apps/src/blockly/addons/cdoFieldDropdown.js

```diff
@@ -32,6 +32,33 @@ export default class CdoFieldDropdown extends GoogleBlockly.FieldDropdown {
     }
   }
 
+  /**
+   * Sets the field's state based on the given state value. Should only be
+   * called by the serialization system.
+   *
+   * @param state The state we want to apply to the field.
+   * @override because different labs store `state` in different ways.
+   * For music lab, `state` is the value of the field.
```

Comment:
nit: Can we update this to clarify that this isn't a "music lab" thing so much as it is a JSON serialization thing?